config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  get "/users" , to: "users#index",  as: "users"
+  get "/users/:id" , to: "users#show" , as: 'user'
+  get "/users/:id/posts" , to: "posts#index", as: "user_posts"
+  get "/users/:id/posts/:id" , to: "posts#show", as: "user_post"
```

Comment:
- [ ] The routes created for the `Post#index` and `Post#show` pages shouldn't have two params named `id`. In later milestones, when you want to retrieve data from the `params` hash, it would be difficult to retrieve the `user's id` from the `params` hash. It is advisable to make it unique. You can consider something like this:

```ruby
 get "/users/:user_id/posts" , to: "posts#index", as: "user_posts"
```

Alternatively, you can consider using the resource routes that Rails provide to create the routes. You can read this [article](https://guides.rubyonrails.org/routing.html#nested-resources) to have a better understanding. 👍 😎

Reply:
@Which command did you use to run the migration file?

Reply:
- You have not implemented the suggestion of the previous reviewer, please use the `user_id` for the `post#index` and `post#show` route in the params hash to differentiate between the user id and post id. Please fix this 🙏